test/inductor/test_torchinductor_opinfo.py

```diff
@@ -214,6 +214,7 @@ def format_op(op):
     "multinomial": {f16, f32, f64},
     "nn.functional.avg_pool1d": {i64},
     "nn.functional.avg_pool2d": {i64},
+    "nn.functional.avg_pool3d": {i64},
```

Comment:
As a follow  up it would be good to find out what's going wrong with `i64` but since this is per-existing I think it's fine for this PR.

Reply:
It gets fixed in https://github.com/pytorch/pytorch/pull/116756/files#diff-57c0f6a344227861850f9d31387ff82cb089898f7472542060349544dade7875L215-L217. Not sure why.